apps/fxc-front/public/static/offline.html

```diff
@@ -0,0 +1,29 @@
bb3HM5oEgBRuXO/8qxYGmwxEDQMEDq1urqXaMREOeCHKSRyUHqmIGtaFRXB2lWCp5+X16LB36AieIkv86wvczrWnTqWHhSBCW6h80HD1nG9H1Ct7DNbbT/xTrktM9BrGRWFwlO0iQ1B00sEm2+N3jRZ1QEir2t5roToaFOgKYKo8R7TsKUwSn0mzEm4lbmhXIU+UYm90kp/83Iq4BFjpvopc5zsM6KUyFFNQcoORwkBBoQhvgtQsQcahKq/SPJQVf5HdoX4vl9ai21kwAapZzjObfz+0NmAdTiVTMAksIR5ynDg56tgAdo3uUgjZpAyeUmjczraA7TZ8IRmubcmGOegB/wBKzT0bueZzcXIN2aRbWoowuueeVzckUXz5MdAonVS0J9Eyd7855HKed4/x0jAXQUOPrUe4+bFAmnX6MLzlFk6i/lubp+kInYmZsBJZwcbmHFqeDOk/B9BsXHbBSvXb275Si53ERWcHAA0TM802qoEEKqFlcIUNK7RHLdNUi65SLdgkQ7OCvuwW87j9PYmDsrQjUUUJo0NpAxpVxPtONnY6J+kNQN1lWS7NZMXDeZQFMVAujn15WhUJNugeKJQN7nutKhrJy8tytZu4GIVt76QgAIVVX0QJLTv1xAEuMiuSV3YlzILkS6yER17vt8vLeLebWiVRHxlRQTXLijWysBghyZLkXgMdmwo0DiOYMaGQ/Cc1E8iE64Olsu1q5sIhwlVPVEe71xQ3yMqs1QKIbl/vl3fXxvQEurQs2MSm8pcJ63Gea+R83qJggQ1+yYogmISQMQZ3n56ZOthHsAMeDA/X0lg0A9J19HcYdOwH9iP4sFJeb4UNxzrUngGrmyYSNw/aJaabU8qo8Lw3HIyLDg44/zavkxPr4kVBAu5eAKASPVlNPK3BXBoqXYoF3xqu92AAjzfEXAgWZ9T1b8lDHBPCKcA4W9F/L9zdTRtDrfZTK3UFZltf0Xn9tmdI62eaXqWIMiJwwlBK7c42xu0soARhFRYnI4h5U3GwHO0Hb8ItpGbvWPhuJAbZ6Y+5gwvqRNoqEU2pQokwKGzsTTZUGrHc7LfjXxevI7hiAK55jMM5MDij5jrECuBTyNLYCTF4uFlYVvXErsZCrutoCe2n7F779UcU/v2QsDnNCCEQohiYEgZzGPiXdFjqOwyHCmdGr4RYIWQXONjiF8ed8yjp6msXpBWf8eEwbatN0ixKtXz25CKqwp1mt7ph12OM3g+AG/HnQIo1MRTFvthfEL2OC8wpi3+wxA9D6jAsl5fD+z+JnP0WZXLW5NFCge51DbYX7/h7zvd7hCYUBSkJ9dmLtTJmfxYrxiufBRJb3IMIvlfhbSaXJisQpjGmxELF3lMU/W55gIUDL+vZqILXF0SxAWHE57tBfglzHgGFYr6ZA6tq/tABcXEaovsLcqcOB1Lu9Hq3tZzs+8wN5znWSpmLnZFpMTKYDff4Y/e81wBjLP18kARA6xU+A9ZYhmWfpII2RizHDoxaJ5Q9+t40kPXTcUntJXWYwWMGE8wscXWIXS5mF0QxheumnFXV5l7vEdwzWEYwwpoAyr85qjdBzxqqGA0FScomWO5XVeJW5h7+P72sDPe6ldqFIRBDnP72zH+fwmBscjXaz+oc87NbgEUTo3pEgo8eOePQUQonr5IAXlh7sTTTAfJHBtWHJW8TP9Ex8v9ywv3eZjh4iDz73MhPEtIzYJEE5W50+ac1bksrFCTUecEpDMNMBoKW4in+GB5vJzXcnjcVNInM4p2JBh0drHn32Sx+BlRdi5OkQnoB8UXQKEKJ2hmNJDagGEMgBClA5WLLoETp1ck1HLDj/TfCYMNP28LsfGHyOORKDeZiaM7xrZ2nIIzTVOGuW1HTBravGxpURiDYymeq9FLu1kPZuf7XIei9+rdJTJ3l5Edf3K31FEzLM5YKGaolMd8FAGJ5kwQDwwUOS4Qfn56pQACFFeaP0oN4K5kwkD1cQ/fgoI4xiP3b8yYbxoVrxCAaFjwNFRAGnT+OA9T3drKiBofaRpOEfL4TQyNlDYL+axuIXHZLVS9ieywYz91yxW3h9RuGPA6GbFzwVFjEtkOxYnApAYUOqIZSm9phtOQ1aRZ8JYz8/4F3y8bpLA8SwD46s8doPO4Gtbv909karjEJd+/j4aG6NmFfQ+Y+XSAZ16TPBdKO2/y+Oxkcfj3C6BAiPEv/HzP8jPf9RwjG5rXOVwCcq4dshsWwsYkwmQHLDkgCAFhBhgYgaFphK7EHpxI798ivcPdfF8IO6neby+zsTxCxoffk3GlNl2uIQGigUIOabIpuEi5HCQRoBzNB3xFht6b2/k/QjP+/qKo5zDr63A3A3xM6Ob1xbe/5Of/QnHdEvkx1DlgiWXS1DGNSYEilMJEEqIWzHOQTWAQzVA1XQ4CrT4C/mZb6qAsob3ywPPIXFez/AOE+4hs5qT4xsiBYzCAUknAg5ybPUWJF7Yf59ZECz3sQQ1s/rvK2ikTzmsX+AoK3lczqeRMBE861beX6mscAO8/4RGKpR0KJ31l9IF6ijSRYLwT0kZ0lMJkFywUCY3yBHJKMBBKKSjKGLHGMDiA7AghH477ycDxNcMPFPhOMQ6jm7SCYgjXiGDUDalvZcQ54gF/bUVB2pVpuELKrAgA/A3vB9SVrhOwF/TqSHq5BB6rt5wymvzni6AeIAoqPuEKpogp2k6x6HkLG+F9v7LEgs5+kWH4p5jT0EPLQZ9kfv1fFMhE6oFsvzHicR9dxxgT4ZuUGSITKetuvuZ6lEYq9gdWiEaNY9j4OmElHmzKo9zQpKfRx/iAkTjwyqI0k6zlMWvYZ4hlOPfdBYlCw4v/CMUG9VxREnKBEEdH0SKM5zWniG91ECniJg6Y7VXcwHinfMCKPscYLQzOVpIPLBZcymzZJ1q+A0HDN5xSuG1ol8n4NeYLGKfOh2merRf9ZQYwOnt//fWnB6C6W16C2//J8AA8rWjFYGIlNoAAAAASUVORK5CYII=">
```

Comment:
Can we use an external image here and cache it at install time ?
Would be easier to inspect that base64